### PR TITLE
task_6 and task_7

### PR DESCRIPTION
**Pull request by ashchavinska**

### CODE REVIEW

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 1]
```diff
@@ -97,4 +102,44 @@ public void verifyLoginWithWrongPassword() {
...
+        HomePage homePage = loginService.login(user.getEmail(), user.getPassword());
+
+        NewsPage newsPage = homePage.getFooterMenu().openNewsPage();
+        Assert.assertEquals(newsPage.getPageTitle(), "News", "News page doesn't open");
+
+        NewsPageItem newsItems = newsPage.pickNews(0);
+        String titleFromNewsPage = newsItems.getTitle();
+
+        ArticlePage articlePage = newsItems.clickNews();
+        Assert.assertEquals(articlePage.postYouCommBtn(), "POST YOUR COMMENT", "Article page doesn't open");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: the verify and the message do not match

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 2]
```diff
@@ -97,4 +102,44 @@ public void verifyLoginWithWrongPassword() {
...
+    @Test(description = "06")
+    @MethodOwner(owner = "ashchavinska")
+    public void verifyArticleName() {
+        UserService userService = new UserService();
+        User user = userService.getUser();
+        LoginService loginService = new LoginService();
+        HomePage homePage = loginService.login(user.getEmail(), user.getPassword());
+
+        NewsPage newsPage = homePage.getFooterMenu().openNewsPage();
+        Assert.assertEquals(newsPage.getPageTitle(), "News", "News page doesn't open");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: the verify and the message do not match

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 3]
```diff
@@ -97,4 +102,44 @@ public void verifyLoginWithWrongPassword() {
...
+    @Test(description = "07")
+    @MethodOwner(owner = "ashchavinska")
+    public void verifySearchingProcess() {
+        UserService userService = new UserService();
+        User user = userService.getUser();
+        LoginService loginService = new LoginService();
+        HomePage homePage = loginService.login(user.getEmail(), user.getPassword());
+
+        NewsPage newsPage = homePage.getFooterMenu().openNewsPage();
+        Assert.assertEquals(newsPage.getPageTitle(), "News", "News page doesn't open");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: the verify and the message do not match

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 4]
```diff
@@ -97,4 +102,44 @@ public void verifyLoginWithWrongPassword() {
...
+
+        final String search = "iPhone";
+        List<NewsPageItem> searchRes = newsPage.searchNews(search);
+        Assert.assertTrue(searchRes.size()>0, "Search result is fail");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: use method isEmpty()

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 1 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 2 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 3 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 4 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**